copyright

```diff
@@ -450,6 +450,12 @@ Copyright: Vilhelm (https://github.com/Vilhelm16)
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (under the same license) 
 
+Files: sounds/thrasher.wav* 
```

Comment:
```suggestion
Files: sounds/thrasher.wav
```
The `*` glob matcher would be needed at the end if you had created e.g. `thrasher.wav1`, `thrasher.wav2`, etc. If you had made e.g. `thrasher~00.wav`, `thrasher~01.wav`, then you would attribute that as `sounds/thrasher*` or `sounds/thrasher*.wav` (since there are more than one and they match that pattern).